tests/taskcluster/tasks/marionette_js_tv_tests.yml

```diff
@@ -0,0 +1,45 @@
+# Chunking task for marionette js tests...
+---
+reruns: 4
+task:
+  metadata:
+    name: 'Marionette JS tests for tv'
+    description: Run entire suite of marionette js tests for tv
+
+  workerType: '{{workerType}}'
+  provisionerId: '{{provisionerId}}'
+  scopes:
+    # Source caches contains various sources (including a tarball of gaia source
+    # tree)
+    - docker-worker:cache:gaia-misc-caches
+    - docker-worker:image:quay.io/mozilla/gaia-taskenv:*
+
+  payload:
+    image: quay.io/mozilla/gaia-taskenv:0.8.9
```

Comment:
We're likely going to roll out gaia-taskenv:0.8.10 very soon. Before committing, check other YML files in the same folder to see if they're using 0.8.10 instead.